### PR TITLE
Add fast-xml-parser

### PR DESCRIPTION
**Pull request by amitguptagwl**
### Reason to add

* Faster than any other pure JS implementation
* don't loose any information in reverse transformation. (xml -> js -> xml)
* can be used in NPM package, browser, or CLI

### CODE REVIEW

**Comment by amitguptagwl** [comment]
Any update?

**Comment by sindresorhus** [comment]
Please read and adhere to the [contributing guidelines](../blob/master/contributing.md). Thank you 🙌

**Comment by amitguptagwl** [comment]
@sindresorhus Here is the comment against each guideline. There is small change in PR as the previous description was starting with letter "A". I believe it is following all the guidelines. I would be interested to know if any particular point is missed by me or I didn't follow correctly.

Ensure your pull request adheres to the following guidelines:

> - If you just created something, wait at least 30 days before submitting. This is to give it some time to mature and ensure it's not just a publish-and-forget type of project. 

It was published 1 year back

> - Keep in mind that the list is very mature by now, so the bar of getting something accepted is high. Only submit something unique and generally useful. The world (and this list) doesn't need yet another ORM or framework. For CLI tools, the bar is especially high, and unless it's something very awesome, I would suggest submitting to awesome-cli-apps instead.

Agree. Hence raised the PR

> - If you submit a project that is similar to an existing project in the list, argue how it's better.

Hence mentioned the reason while raising the PR.

> - Search previous suggestions before making a new one, as yours may be a duplicate.

It is similar to `xml2js` which was already added before. But better in terms of speed and features. I've mentioned in PR description.

> - Suggested packages should be tested and documented.

Yes it is.

> - Make an individual pull request for each suggestion.

Hence created a separate PR

> - Use the following format: [package](link) - Description.

Done

> - Additions should be added to the bottom of the relevant category.

Done

> - Link to the GitHub repo, not npmjs.com.

Done

> - Keep descriptions short and simple, but descriptive.

Done

> - Don't mention Node.js in the description as it's implied.

Done

> - Start the description with a capital and end with a full stop/period.

Done

> - Don't start the description with A or An.

Just edited the PR

> - Check your spelling and grammar.

Done
> - Make sure your text editor is set to remove trailing whitespace.

Done

>- The pull request should have a useful title and include a link to the package and why it should be included.

As the link was already the part of PR, I didn't it's need. However here is the link: https://github.com/NaturalIntelligence/fast-xml-parser

> - New categories or improvements to the existing categorization are welcome, but should be done in a separate pull request.

Done

File: readme.md [hunk 1]
```diff
@@ -552,6 +552,7 @@ Just type [`node.cool`](https://node.cool) to go here ✨
 - [Stylecow](https://github.com/stylecow/stylecow) - Parse, manipulate and convert modern CSS to make it compatible with all browsers. Extensible with plugins.
 - [js-yaml](https://github.com/nodeca/js-yaml) - Very fast YAML parser.
 - [excel-stream](https://github.com/dominictarr/excel-stream) - Streaming Excel spreadsheet to JSON parser.
+- [fast xml parser](https://github.com/NaturalIntelligence/fast-xml-parser) - Very fast XML to Js Obj/JSON or Nimn and JSON/Js obj to XML converter. It can validate XML syntax and can work from CLI as well.
```
Comment (sindresorhus) [inline readme.md]: The name should be identical to the npm package name. The description also needs work. Try to be as short as possible.

Comment (amitguptagwl) [reply readme.md]: Thanks for your review. Done :+1:

**Comment by amitguptagwl** [comment]
any update?

**Pull request merged**